### ISSUE TITLE
Exercises Day 2

### ISSUE THREAD

**Issue by constantinpape** [constantinpape/training-deep-learning-models-for-vison#4]
Conceptually the exercises for day 2 are ok.
However there are still too many training runs, so it would be good to reduce the scope of the notebooks.

For example in exercise 1, we should just add a single model deeper model and add some tips on how to construct it.
Also, include batchnorm or instance / group norm in the task here
Put the adaptive average pooling in an advanced task.

Exercise 2: ok if we have batch norm in the first exercise.

Exercise 3: don't need the combination with data augmentation, just train one more architecture from torch vision.

**Comment by constantinpape** [comment]
Re exercise 3: finding another model that works is very difficult -> remove that part of the exercise.
